<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::EnumsTab</class>
 <widget class="QWidget" name="GammaRay::EnumsTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_8">
   <item>
    <widget class="QLineEdit" name="enumSearchLine"/>
   </item>
   <item>
    <widget class="QTreeView" name="enumView">
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
